src/MicrosoftTeams.ts

```diff
@@ -417,25 +442,132 @@ namespace microsoftTeams
         {
             ensureInitialized(frameContexts.content, frameContexts.settings, frameContexts.remove);
 
+            // Open an authentication window with the parameters provided by the caller
+            openAuthenticationWindow(authenticateParameters.url, authenticateParameters.width, authenticateParameters.height);
+
+            authParams = authenticateParameters;
+        }
+
+        function closeAuthenticationWindow(): void
+        {
+            // Stop monitoring the authentication window
+            stopAuthenticationWindowMonitor();
+
+            // Try to close the authentication window and clear all properties associated with it
+            try
+            {
+                if (childWindow)
+                {
+                    childWindow.close();
+                }
```

Comment:
Looks like the child window is trying to close itself as well as the the parent trying to close the child. Should we be content with one or the other?